public/app/percona/dbaas/components/DBCluster/DBCluster.service.ts

```diff
@@ -68,6 +69,16 @@ export abstract class DBClusterService {
     );
   }
 
+  static async getDBClusterSecrets(kubernetesClusterName: string): Promise<DBClusterSecretsAPI> {
+    return apiManagement.post<DBClusterSecretsAPI, any>(
```

Comment:
If possible create a type for the request to prevent additional `any`s in the code.

Reply:
fixed